src/Forms/test/Xamarin.Forms.Xaml.UnitTests/Issues/Bz45891.xaml.cs

```diff
@@ -43,11 +43,11 @@ public void TearDown()
 				Device.PlatformServices = null;
 			}
 
-			[Ignore("Temporarily ignore: https://github.com/xamarin/Xamarin.Forms/issues/13815")]
 			[TestCase(true)]
 			[TestCase(false)]
 			public void LookForInheritanceOnOpImplicit(bool useCompiledXaml)
 			{
+				((MockPlatformServices)Device.PlatformServices).RuntimePlatform = Device.iOS;
```

Comment:
This is needed because the old way would fall back to iOS, so we just explicitly say that it will be iOS.